DigitalLearningSolutions.Data/Models/DelegateUpload/DelegateTableRow.cs

```diff
@@ -143,5 +187,12 @@ public bool MatchesDelegateUser(DelegateUser delegateUser)
 
             return (delegateUser.EmailAddress ?? string.Empty) == Email;
         }
+
+        private bool IsEmailFormatValid()
+        {
+            var whitespace = Email!.Any(char.IsWhiteSpace);

```

Comment:
Super minor, but I'd expect something called "whitespace" to be = "   " or similar. Maybe hasWhitespace?